main.go

```diff
@@ -41,9 +43,11 @@ func main() {
 	localAddr.IP = ip
```

Comment:
Re. my comment on hard-failing if the download directory doesn't exist, I tried the following patch:
```
$ git diff
diff --git a/main.go b/main.go
index f57d340..f0b5a8c 100644
--- a/main.go
+++ b/main.go
@@ -4,6 +4,7 @@ import (
        "flag"
        "log"
        "net"
+       "os"
 )
 
 // TODO handle expired Slack RTM sessions (e.g. after standby/resume)
@@ -41,6 +42,12 @@ func main() {
                log.Fatal("Invalid IP address to listen on")
        }
        localAddr.IP = ip
+       if *fileDownloadLocation != "" {
+               dInfo, err := os.Stat(*fileDownloadLocation)
+               if err != nil || !dInfo.IsDir() {
+                       log.Fatalf("Missing or invalid download directory: %s", *fileDownloadLocation)
+               }
+       }
        log.Printf("Starting server on %v", localAddr.String())
        server := Server{
                LocalAddr:            &localAddr,
```